### DISCUSSION TITLE
Integration of Microsoft Exchange with Full Calendar

### DISCUSSION THREAD

**Discussion by kkathman** [davish/obsidian-full-calendar#461] [Feature Requests]
### ⚠️ Please check that this feature request hasn't been suggested before.

- [X] I searched previous [Ideas in Discussions](https://github.com/davish/obsidian-full-calendar/discussions/categories/feature-requests) didn't find any similar feature requests.
- [X] I searched previous [Issues](https://github.com/davish/obsidian-full-calendar/issues) didn't find any similar feature requests.

### 🔖 Feature description

Like many, I work within a large organization (a University) that used Microsoft Exchange. I have been able to sync this calendar real time with Apple Calendar and NotePlan3 is able to REAL TIME show a timeline/calendar that shows ALL corporate events in a daily calendar.

I was unable to do this with Day Planner and was hoping that it could be accomplished via Full Calendar.

On my Mac, I can open up Apple Calendar and see ALL Exchange calendar entries with no problems. I can bring up the Apple Calendar and see ALL Exchange event's I can NOT see any of these in Full Calendar, however.  

Can you remediate this issue? It may be similar to Goole Coalendar in that Oaith is involve.

### ✔️ Solution

I want to see ALL Outlook Exchange items show up on the calemdar

**Comment by sonihemish** [comment]
Plus one onf the exchange integration ! 
I also have icalbuddy installed and am able to pull the events through templater .. however a calendar view would be great using icalbuddy. so that I can be selective of which meetings i need to create notes from.

**Comment by thederekmast** [comment]
I also would love this.
